### ISSUE TITLE
[Improvement] Commandline administration

### ISSUE THREAD

**Issue by AiyionPrime** [luhsra/MinimaxSimulator#57]
I think it would be a huge improvement if one could control loading the project and memory chunks, running the project as well as exporting memory via commandline parameters.
This would greatly improve reliable testing as well as better documentation approaches and reproducibility.

Good starters would be:
- one project file to load
- several memory chunks to import and their target positions
- several export memory chunks and their positions as well

Running the project would be an implicit step after loading and before exporting.

**Comment by prohde** [comment]
So you would like to load a project, load some data into a specified position of the simulated memory, run the project, export (part of) the simulated memory all with a single command-line call?

**Comment by AiyionPrime** [comment]
Yes, I think that would make the simulator scriptable and allow to write some sort of tests for its projects.

**Comment by prohde** [comment]
I'd suggest to use [picocli](https://picocli.info/) as command line argument parser. It seems to be relatively easy to use. At this point it is actively maintained. What are your thought? Would you rather go with [Apache Commons CLI](https://commons.apache.org/proper/commons-cli/)? In general, the Apache Java libraries seem to be not much maintained in the last years.

**Comment by AiyionPrime** [comment]
I'm sorry to have no strong opinions on cli-libraries, as I haven't written proper java code in ages.
But picocli appears to be a very well suited candidate and I have a thing for keeping the code short :+1:

**Comment by prohde** [comment]
Did you already check the PR draft for a CLI version?

**Comment by AiyionPrime** [comment]
No not yet, but will tomorrow.
I didn't occure to me, yet - sorry.
